### PR TITLE
refact: utlize wait and shutdown shell , normalize tests and docker comp

### PR DESCRIPTION
**Pull request by moogacs**
this PR 
- introduces `common.sh` file to utilize the `wait_weaviate()` and `shutdown()` shell functions  
- remove redundant impl. for wait_weaviate() and shutdown()
- unify the test docker volumes to be created and  cleaned after the test

### CODE REVIEW

File: replication_importing_with_backup.sh [hunk 1]
```diff
@@ -73,9 +53,9 @@ if [[ $exit_code_imp_1 != 0 && $exit_code_imp_1 != 137 ]] || \
...
   echo "Could not import/backup"
-  shutdown
-  exit 1
+  # shutdown
+  # exit 1
 fi
 
 echo "Passed!"
-shutdown
+# shutdown
```
Comment (dirkkul) [inline replication_importing_with_backup.sh:61]: why commented out?

File: common.sh [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+    
+  compose_files=$(ls apps/weaviate/docker-compose-*.yml)
+  for file in $compose_files; do
+    docker compose -f "$file" down --remove-orphans
+  done
+  
+  rm -rf apps/weaviate/data* || true    
+  rm -rf workdir
+}
+trap 'shutdown; exit 1' SIGINT ERR
```
Comment (jfrancoa) [inline common.sh:38]: Something that would be missing with this implementation of shutdown it's:
```
echo "Showing logs:"
docker compose -f apps/weaviate/docker-compose-replication.yml logs weaviate-node-1 weaviate-node-2 weaviate-node-3
```
which is pretty useful for debugging failed cases, although I have been thinking on enabling logging for all jobs and storing them in Artifacts, maybe now it's the moment.

File: replication_importing_with_backup.sh (hunk 1 above)
Comment (moogacs) [reply replication_importing_with_backup.sh:61]: good catch, that was me testing locally something  🤦🏼‍♂️

File: common.sh (hunk 2 above)
Comment (moogacs) [reply common.sh]: good idea

**Review by jfrancoa — APPROVED** [review]
Thanks for addressing the logs comment.

**Pull request merged**